camviewer_ui_impl.py

```diff
@@ -2026,6 +2026,24 @@ def connectCamera(self, sCameraPv, index, sNotifyPv=None):
         # Get camera configuration
         self.getConfig()
 
+        # Check the expected size against the count to generate warnings
+        first_image_count = len(self.camera.value)
+        expected_count = self.rowPv.value * self.colPv.value
+        if self.isColor:
+            expected_count *= 3
```

Comment:
Any image that is color is always RGB?   No funny CMYK or otherwise?

Reply:
I checked into this a little bit, there are a lot of instances of `3` hard-coded into the sip file, so it seems like it. There are direct references to RGB as well.

In general (separate to the source code) I expect CMYK to only be used for printers and for all computer screen graphics to use RGB due to additive vs subtractive color theory.

Reply:
I'll admit I am not an expert on color theory, I just saw a magic number and had to question it haha.